regulations/static/regulations/js/source/views/header/preamble-head-view.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var PreambleHeadView = Backbone.View.extend({
+  el: '.preamble-header',
+
+  events: {
+    'click .read-proposal': 'readProposal',
+    'click .write-comment': 'writeComment'
+  },
+
+  initialize: function() {
+    this.$read = $('#preamble-read');
```

Comment:
We don't seem to use `this.$read` or `this.$write`. Do we need them in this view?